### DISCUSSION TITLE
V5 stops working when using createRoot react 18

### DISCUSSION THREAD

**Discussion by masterbater** [remix-run/react-router#8768] [General]
Guys is there anyone of you experience this. ReactDom.render routing works but when using the createRoot path is changing but the component is not being mounted.

**Comment by khanilov** [comment]
Probably you have `StrictMode` enabled. There is known issue with it:
https://github.com/remix-run/react-router/issues/7870

**Comment by ChineseBread** [comment]
me too, i use v6 and React 18 even give me a warning says that react 18 no longer support ReactDOM.render() anymore

**Reply by masterbater** [reply]
I migrated to v6 and I dont encounter this issue

**Reply by ChineseBread** [reply]
when i use `const navigator = useNavigate()` react give me the warning

**Reply by seda1094** [reply]
I am getting 
`Routes not found via ../../node_modules/react-router-dom-v5-compat/dist/index.js -> ../../node_modules/react-router/esm/react-router.js  import/named`

and the same for `useNavigate`

**Comment by sudharsteyn** [comment]
Yeah, I'm using react-router-dom 5.2.0 with createRoot if I'm navigating pages it changes only the URL the component not displaying

Any solution for that or should I update the react-router-dom?

**Reply by yohanelly95** [reply]
hey did you find a solution for this?

**Reply by sudharsteyn** [reply]
Downgrade your react version to 17 or check the doc how to use [createRoot](https://react.dev/reference/react-dom/client/createRoot)
